jerry-main/main-unix.c

```diff
@@ -485,6 +491,38 @@ main (int argc,
                         "Ignoring 'exec-snapshot' option because this feature isn't enabled!\n");
       }
     }
+    else if (!strcmp ("--save-literals-list-format", argv[i])
+             || !strcmp ("--save-literals-c-format", argv[i]))
+    {
+      if (++i >= argc)
+      {
+        jerry_port_log (JERRY_LOG_LEVEL_ERROR, "Error: no file specified for %s\n", argv[i - 1]);
+        print_usage (argv[0]);
+        return JERRY_STANDALONE_EXIT_CODE_FAIL;
+      }
+
+      if (jerry_is_feature_enabled (JERRY_FEATURE_SNAPSHOT_SAVE))
+      {
+        is_save_literals_mode = true;
+        is_save_literals_mode_in_c_format_or_list = !strcmp ("--save-literals-c-format", argv[i - 1]);
```

Comment:
I've updated the PR according to your review, but there is another occurence the mentioned comparison at [main-unix.c:445](https://github.com/jerryscript-project/jerryscript/blob/master/jerry-main/main-unix.c#L445) in save-snapshot section. It could be modified in a follow-up patch.